solidity/test/BondedECDSAKeepTest.js

```diff
@@ -1865,6 +1865,11 @@ contract("BondedECDSAKeep", (accounts) => {
   }
 
   async function depositForBonding(member1Value, member2Value, member3Value) {
+    const beneficiary = accounts[4]
```

Comment:
Can we define `beneficiary` on the top of the file, where we define the `bondCreator`, `members`, `authorizers` and all the rest of the roles?

Reply:
Sure, done.